odoo/cli/server.py

```diff
@@ -136,6 +138,11 @@ def main(args):
         for db_name in preload:
             try:
                 odoo.service.db._create_empty_database(db_name)
+            except ProgrammingError as err:
+                if err.pgcode == errorcodes.INSUFFICIENT_PRIVILEGE:
+                    _logger.warning("Could not determine if database exists: %s" % err)
```

Comment:
@odony if I can comment - a `_logger.warning()` is better then a `_logger.info()`. Info is, well, for information, but failing to determine a database is more then information to me. Just my opinion.